README.md

```diff
@@ -1,32 +1,68 @@
 ![Mollie](https://www.mollie.nl/files/Mollie-Logo-Style-Small.png)
 
-# Mollie API client for Node #
+JavaScript client for Mollie's API.
```

Comment:
Yeah, node is even in the name, it should say that it's for node because some people might think you can use it client side (like db on firebase)